docs/source/notebook.rst

```diff
@@ -150,7 +156,8 @@ This long string is the kernel's ID which is sufficient for getting the
 information necessary to connect to the kernel.  You can also request this
 connection data by running the ``%connect_info`` :ref:`magic
 <magics_explained>`. This will print the same ID information as well as the
-content of the JSON data structure it contains.
+content of the JSON data structure contained in the associated kernel file
```

Comment:
Let's call this a 'connection file', which is the term we use for it elsewhere - kernel file could mean other things.